src/Symfony/Component/DependencyInjection/Argument/TaggedIteratorArgument.php

```diff
@@ -16,6 +16,7 @@
  *
  * @author Roland Franssen <franssen.roland@gmail.com>
  */
+#[\Attribute(\Attribute::TARGET_PARAMETER)]
```

Comment:
I think we should create new classes in the Attribute namespace instead:
- the `needsIndexes` should not be exposed (it's mostly here to build locators)
- the index and priority-related arguments can be removed (on the attribute classes) since they won't be needed once #40248 is accepted.